Description: don't specify VIKING_DOC

 - don't specify VIKING_DOC since we don't have doc sub directory
 - GTK_DOC_CHECK cause parse error

Forwarded: no
Last-Update: 2013-10-23

Index: viking-1.4.2/configure.ac
===================================================================
--- viking-1.4.2.orig/configure.ac	2013-10-23 21:50:46.822460778 +0900
+++ viking-1.4.2/configure.ac	2013-10-23 22:03:31.030122042 +0900
@@ -6,7 +6,6 @@
 AM_INIT_AUTOMAKE([dist-bzip2 dist-zip subdir-objects])
 dnl AC_CONFIG_SRCDIR([src/main.c])
 AC_CONFIG_HEADERS([src/config.h])
-GTK_DOC_CHECK(1.0)
 
 # Checks for programs.
 AC_PROG_CC
@@ -342,8 +341,6 @@
 ACLOCAL_AMFLAGS="\${ACLOCAL_FLAGS}"
 AC_SUBST(ACLOCAL_AMFLAGS)
 
-AM_CONDITIONAL([VIKING_DOC], test x${enable_gtk_doc} = xyes)
-
 # Configuration
 AC_CONFIG_FILES([
 		Makefile
@@ -357,11 +354,6 @@
 		win32/installer/Makefile
 		win32/installer/pixmaps/Makefile
 		win32/installer/translations/Makefile])
-AM_COND_IF([VIKING_DOC],
-	[AC_CONFIG_FILES([
-		doc/Makefile
-		doc/reference/Makefile
-		doc/examples/Makefile])])
 
 AC_OUTPUT([
 	   viking.spec
Index: viking-1.4.2/Makefile.am
===================================================================
--- viking-1.4.2.orig/Makefile.am	2013-06-01 07:01:35.000000000 +0900
+++ viking-1.4.2/Makefile.am	2013-10-23 21:51:01.934450885 +0900
@@ -1,8 +1,4 @@
-# Optionally build documentation bits
-if VIKING_DOC
-	DO_DOCS = "doc"
-endif
-SUBDIRS = src test data po win32 help $(DO_DOCS)
+SUBDIRS = src test data po win32 help
 
 INTLTOOL = \
 	intltool-extract.in \
